### PR TITLE
Update changelog for 0.7.0

### PR DESCRIPTION
**Pull request by matthiasgeihs**

### CODE REVIEW

**Comment by ggwpez** [comment]
I dont know how you did it before, but one tip on how to get all merge commits:  
`git log --reverse --merges v0.6.0..origin/dev`  
since the last release.

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,56 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.7.0] Ganymed - 2021-07-09 [:warning:]
```
Comment (ggwpez) [inline CHANGELOG.md:9]: The 0.7.0 link is dead.  
You can add the links in the bottom, should be something like:  
`https://github.com/hyperledger-labs/go-perun/compare/v0.6.0...v0.7.0`.

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,6 +6,56 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
```
Comment (ggwpez) [inline CHANGELOG.md:7]: Please update the unreleased link.

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,6 +6,56 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
 
+## [0.7.0] Ganymed - 2021-07-09 [:warning:]
+Virtual channels. And some other additions.
+
+### Added :sparkles:
+- **Virtual Channels** ([#83](https://github.com/hyperledger-labs/go-perun/pull/83), [#114](https://github.com/hyperledger-labs/go-perun/pull/114), [#119](https://github.com/hyperledger-labs/go-perun/pull/119), [#123](https://github.com/hyperledger-labs/go-perun/pull/123)): Go-perun now supports virtual channels. A virtual channel is a channel that is funded and settled completely off-chain and therefore does not incur any on-chain transaction fees.
+- **Generic event subscription** ([#36](https://github.com/hyperledger-labs/go-perun/pull/36), [#86](https://github.com/hyperledger-labs/go-perun/pull/86), [#89](https://github.com/hyperledger-labs/go-perun/pull/89), [#94](https://github.com/hyperledger-labs/go-perun/pull/94)): In preparation for implementing a reorg-resistant event subscription, we implemented a generic event subscription that can be used across the whole library.
+- **Tutorial announcement** ([c8cff7cb](https://github.com/perun-network/go-perun/commit/c8cff7cbbe97e5b2127e7109483026d1b3938453)): We now have a developer tutorial.
```
Comment (ggwpez) [inline CHANGELOG.md:15]: I think the link to the actual tutorial would be nice here.

File: CHANGELOG.md [hunk 4]
```diff
@@ -6,6 +6,56 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- **Current state in HandleUpdate** [#33](https://github.com/hyperledger-labs/go-perun/pull/33): The update handler now receives the current channel state as a parameter.
+- **Funder usability** [#74](https://github.com/hyperledger-labs/go-perun/pull/74): Streamlined Eth funder setup.
+- 2021 updates ([2212847d](https://github.com/perun-network/go-perun/commit/2212847de68d683865427c7e11abd48b589f90ee)),
+Update generate script ([3f81e47c](https://github.com/perun-network/go-perun/commit/3f81e47cfe436ec42ed4ae9d91742e58f64a0013)),
+Update links to HLL ([#6](https://github.com/hyperledger-labs/go-perun/pull/6)),
+Update security disclaimer ([#14](https://github.com/hyperledger-labs/go-perun/pull/14), [#51](https://github.com/hyperledger-labs/go-perun/pull/51)),
+Delete gitlab templates ([#24](https://github.com/hyperledger-labs/go-perun/pull/24)),
+Dependency update ([#30](https://github.com/hyperledger-labs/go-perun/pull/30)),
+Document parameters of NewLedgerChannelProposal ([#43](https://github.com/hyperledger-labs/go-perun/pull/43))
```
Comment (ggwpez) [inline CHANGELOG.md:34]:
```suggestion
Document parameters of NewLedgerChannelProposal ([#43](https://github.com/hyperledger-labs/go-perun/pull/43)),
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -6,6 +6,56 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+CI speed-up ([#44](https://github.com/hyperledger-labs/go-perun/pull/44)),
+Refactor Eth channel errors [#88](https://github.com/hyperledger-labs/go-perun/pull/88),
```
Comment (ggwpez) [inline CHANGELOG.md:36]:
```suggestion
Refactor Eth channel errors ([#88](https://github.com/hyperledger-labs/go-perun/pull/88)),
```

**Review by ggwpez — COMMENTED** [review]
I like the new `Security` section, nicely done @matthiasgeihs

File: CHANGELOG.md (hunk 1 above)
Comment (sebastianst) [reply CHANGELOG.md:9]: yes, please adjust the bottom list of links

File: CHANGELOG.md [hunk 6]
```diff
@@ -6,6 +6,56 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- **Named errors** ([429a8934](https://github.com/perun-network/go-perun/commit/429a8934666db35d99186d874c23f1102e78750d), [#10](https://github.com/hyperledger-labs/go-perun/pull/10), [#11](https://github.com/hyperledger-labs/go-perun/pull/11), [#26](https://github.com/hyperledger-labs/go-perun/pull/26), [#34](https://github.com/hyperledger-labs/go-perun/pull/34), [#80](https://github.com/hyperledger-labs/go-perun/pull/80)): Specific error types help the library user to identify the cause of an error.
+- **Register asset at runtime** [#124](https://github.com/hyperledger-labs/go-perun/pull/124): It is now possible to add assets to the Eth funder at runtime.
+- Gatherer.OnFail ([e3729a6a](https://github.com/perun-network/go-perun/commit/e3729a6a2f0231273d1cb4ee05ef113c335cdb05)),
+Test wallet ([12c78d33](https://github.com/perun-network/go-perun/commit/12c78d33ba339a328d8e5a7b7cf241d9cafca157)),
+GitHub CI ([#3](https://github.com/hyperledger-labs/go-perun/pull/3)),
+ConcurrentT.WaitCtx ([#112](https://github.com/hyperledger-labs/go-perun/pull/112)),
```
Comment (sebastianst) [inline CHANGELOG.md:21]: Please put code-like stuff in backticks (applies to the whole diff).

**Review by sebastianst — CHANGES_REQUESTED** [review]
I think it's also worth mentioning that we changed to [EIP-155 transaction signatures](https://eips.ethereum.org/EIPS/eip-155) in the eth backend with [this commit](https://github.com/hyperledger-labs/go-perun/commit/c37f566d0736726f9397b03c22a8579fada8c7df).

Also please update the release diffs at the bottom of the `CHANGELOG`.

**Comment by matthiasgeihs** [comment]
Addressed all comments, thanks!

**Review by sebastianst — APPROVED** [review]
lgtm

**Review by sebastianst — APPROVED** [review]

**Pull request merged**